packages/components/src/styles/components/side-nav/main.scss

```diff
@@ -8,25 +8,19 @@
 //
 
 .hds-side-nav {
-  position: sticky;
-  top: 0;
-  z-index: 20; // needs to stay above the main content
+  // Testing moving SideNav layout styles to AppFrame sidebar container...
+  // position: sticky;
+  // top: 0;
+  // z-index: 20; // needs to stay above the main content
   width: var(--hds-app-sidenav-width-fixed); // "default" width used by the `SideNav::Base` subcomponent (that is not responsive)
-  height: 100vh;
-  min-height: 100vh;
+  // height: 100vh;
+  // min-height: 100vh;
+  height: 100%;
+  min-height: 100%;
```

Comment:
I'm testing moving the above styles related to the "sticky" layout to app-frame.scss styles for the "sidebar" container. (To give the AppFrame responsibility for the sticky/fixed layout of child components.)